_sass/autumn.scss

```diff
@@ -0,0 +1,67 @@
+// Syntax highlighting for code blocks (light)
```

Comment:
do we use this?

Reply:
not yet, but it's 100% something worth future proofing in case we ever want a formatted code block